usage/jsgui/src/main/webapp/assets/js/view/entity-config.js

```diff
@@ -181,7 +181,7 @@ define([
                     if (!$(this).closest("#config-table").length) return;
                     var text = $(this).attr('copy-value');
                     if (!text) text = $(this).closest('.floatGroup').find('.value').html();
```

Comment:
Would be easier to use .text() instead of trying to unescape the html. Also `_.unescape` works for a limited html escapes.